js/stateless.js/src/idls/light_system_program.ts

```diff
@@ -194,23 +216,36 @@ export type LightSystemProgram = {
                     name: 'accountCompressionAuthority';
                     isMut: false;
                     isSigner: false;
+                    docs: [
+                        'This pda is used to invoke the account compression program.',
+                    ];
                 },
                 {
                     name: 'accountCompressionProgram';
                     isMut: false;
                     isSigner: false;
+                    docs: ['Merkle trees.'];
                 },
                 {
                     name: 'solPoolPda';
                     isMut: true;
                     isSigner: false;
                     isOptional: true;
+                    docs: [
+                        'Sol pool pda is used to store compressed sol.',
```

Comment:
doesn't actually store compressed sol but native sol

Reply:
Joa both not wrong, the native sol that has been compressed.
Makes sense to be explicit let me fix it in https://github.com/Lightprotocol/light-protocol/actions/runs/9754900751/job/26922536668?pr=931 since the tests are already green here.